django_sirene/importers.py

```diff
@@ -55,46 +83,37 @@ def _is_headquarter(self, row):
 
         :param row: dict containing the current row from csv {column: value}
         """
-        return bool(int(row['SIEGE']))
+        return row["etablissementSiege"] == "true"
```

Comment:
Shouldn't we be doing this instead ?
```suggestion
        return row["etablissementSiege"].lower() == "true"
```